code-env/python/desc.json

```
@@ -1,5 +1,5 @@
 {
-  "acceptedPythonInterpreters": ["PYTHON34","PYTHON35","PYTHON36"],
+  "acceptedPythonInterpreters": ["PYTHON34","PYTHON35","PYTHON36","PYTHON37"],
```

```suggestion
  "acceptedPythonInterpreters": ["PYTHON35","PYTHON36","PYTHON37"],
```

Can you make this change, too? Just noticed that [typing](https://docs.python.org/3/library/typing.html) wasn't available until 3.5.